<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WOpenSpace</class>
 <widget class="QWidget" name="WOpenSpace">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>854</width>
    <height>384</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="libUniBtn">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>110</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Libreria Uni</string>
   </property>
  </widget>
  <widget class="QPushButton" name="salottoBtn">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>110</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Salotto</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cucinaBtn">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>110</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Cucina</string>
   </property>
  </widget>
  <widget class="QPushButton" name="scarpieraBtn">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>110</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Scarpiera</string>
   </property>
  </widget>
  <widget class="QPushButton" name="anticameraBtn">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>170</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Anticamera</string>
   </property>
  </widget>
  <widget class="QPushButton" name="balconeBtn">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>170</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Balcone</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
